File: cloud/oauth-server/src/main/java/pers/acp/admin/oauth/repo/ApplicationRepository.java
package pers.acp.admin.oauth.repo;

import pers.acp.admin.oauth.base.BaseRepository;
import pers.acp.admin.oauth.entity.Application;

import java.util.List;

/**
 * @author zhangbin by 2018-1-17 17:44
 * @since JDK 11
 */
public interface ApplicationRepository extends BaseRepository<Application, String> {

    void deleteByIdInAndCovert(List<String> idList, boolean covert);

}

File: cloud/oauth-server/src/main/kotlin/pers/acp/admin/oauth/repo/ApplicationRepository.kt
package pers.acp.admin.oauth.repo

import pers.acp.admin.common.base.BaseRepository
import pers.acp.admin.oauth.entity.Application

/**
 * @author zhangbin by 2018-1-17 17:44
 * @since JDK 11
 */
interface ApplicationRepository : BaseRepository<Application, String> {

    fun deleteByIdInAndCovert(idList: MutableList<String>, covert: Boolean)

    fun findAllByOrderByIdentifyAscAppNameAsc(): MutableList<Application>

}
